extensions/resource-deployment/src/interfaces.ts

```diff
@@ -285,7 +285,7 @@ export interface IOptionsSource {
 export interface OptionsInfo {
 	values?: string[] | azdata.CategoryValue[],
 	source?: IOptionsSource,
-	defaultValue: string,
+	defaultValue: string | azdata.CategoryValue,
```

Comment:
Why are you changing this? Specifying the CategoryValue object here again doesn't seem right - the defaultValue just tells us which value in the list should be the default and the name alone should be enough to pick that out. (Was it not working or something?)